llm_sandbox/kubernetes.py

```
@@ -467,6 +486,48 @@ def _default_pod_manifest(self) -> dict:
+            "runAsUser": 1000,
+            "runAsGroup": 1000,
+            "runAsNonRoot": True,
+            "allowPrivilegeEscalation": False,
+            "readOnlyRootFilesystem": True,
+            "capabilities": {"drop": ["ALL"]},
+        }
+        return {"pod": pod_context, "container": container_context}
+
+    def _strict_pod_volume_mounts(self) -> list[dict[str, str]]:
+        """Return writable mounts needed when the strict profile uses a read-only root filesystem."""
+        mounts = [{"name": "sandbox-tmp", "mountPath": "/tmp"}]
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)